critical-update/feature-flag.tf

```diff
@@ -0,0 +1,19 @@
+data "launchdarkly_project" "critical-updates" {
+  key = "${var.project-key}"
+}
+
+resource "launchdarkly_feature_flag" "remove-static-queries" {
```

Comment:
Is the temporary flag could be needed later?

Reply:
It's not temporary, this is the flag for the update. I just forgot to change the name for something generic (fixed now).